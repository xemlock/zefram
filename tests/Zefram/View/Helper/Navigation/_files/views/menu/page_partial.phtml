<strong><?php echo $this->escape($this->page->getLabel()) ?></strong>
