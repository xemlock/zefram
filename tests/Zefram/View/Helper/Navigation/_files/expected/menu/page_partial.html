<ul class="navigation">
    <li>
        <strong>Partial</strong>
    </li>
</ul>
